app/javascript/src/components/menus/connection_menu.js

```diff
@@ -11,7 +11,8 @@ class ConnectionMenu extends ActivatedMenu {
     super($node, mergeObjects({
       activator_classname: $node.data("activator-classname"),
       container_id: $node.data("activated-menu-container-id"),
-      activator_text: $node.data("activator-text")
+      activator_text: $node.data("activator-text"),
+      activator_icon: $node.data("activator-icon")
```

Comment:
But the issue here is that it can't be controlled by css - any icon injected by css will be read out by a screen reader, so it is not a 'design only' element...  

Maybe it is simply the naming... I've called it icon which implies a 'design' element, but it can essentially be thought of as a 'visible' label and an `accessible` label.

The only other solution I can see to this would be using SVG... ...which may work and would keep presentational things out of the markup as I think I can use an svg url in the `::after' content attribute.

I'll have a play and see what I can come up with...